src/cephadm/cephadmlib/logging.py

```diff
@@ -102,29 +157,85 @@ def filter(self, record: logging.LogRecord) -> bool:
 """
 
 
+_VERBOSE_HANDLERS = [
+    'console',
+    'console_stdout',
+    LogDestination.file.value,
+    LogDestination.syslog.value,
+]
+
+
+_INTERACTIVE_CMDS = ['bootstrap', 'rm-cluster']
+
+
+def _copy(obj: Any) -> Any:
+    """Recursively copy mutable items in the logging config dictionaries."""
+    # copy.deepcopy fails to pickle the config dicts (sys.steerr, etc)
```

Comment:
```suggestion
    # copy.deepcopy fails to pickle the config dicts (sys.stderr, etc)
```
NIT, typo

Reply:
Maybe I was thinking about steak that day. Fixed.